modules/ROOT/pages/sources-reference.adoc

```diff
@@ -0,0 +1,85 @@
+= Mule Sources
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, flow, source, sources, redelivery, policy
+
+Sources are a very important part of any Mule Application. When added to a flow, they serve as the entry point for information,
+linking any external communication protocol with internal Mule processing.
+
+Examples of sources are:
+
+* xref:docs-connectors::http-connector.adoc[HTTP:Listener]
+* xref:docs-connectors::file-connector.adoc[FILE:Directory Listener]
+* xref:docs-connectors::amqp-connector.adoc[AMQP:Consume]
+
+== Configuring Sources
+
+Even though each source implementation will have different configuration parameters, there are some common ones worth mentioning.
+
+=== Redelivery Policy
+
+Each source can be configured with a redelivery policy. That way, an user can add some logic on how to handle messages that arrive at the source multiple times and cause errors.
+
+[IMPORTANT]
+Some sources implemented with the xref:docs-mule-sdk::index.adoc[Mule SDK] have their own logic for a redelivery policy. In those cases, check the documentation for the connector which that source belongs to for more information.
```

Comment:
`have their own logic for a redelivery policy` what do you mean with this?
Also all connectors are made with the SDK

Reply:
Check Pablo La Greca's comment